rero_ils/config.py

```diff
@@ -665,7 +664,7 @@ def _(x):
         search_factory_imp='rero_ils.query:organisation_organisation_search_factory',
         list_permission_factory_imp=can_access_organisation_patrons_factory,
         create_permission_factory_imp=deny_all,
-        update_permission_factory_imp=can_update_organisations_factory,
+        update_permission_factory_imp=is_system_librarian_organisation_record_factory,
```

Comment:
I recall that it was decided that the `organisation` record should not be manipulated by a librarian/sys_librarian. instead it will be managed by rero_centrale or ucl_admins... possibly need to talk to PO about this.

Reply:
I used `deny_all` for all organisation permissions